scripts/parachains_integration_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
(bit nicer for nixos users and more the foolproof way)